content/_data/changelogs/lts.yml

```diff
@@ -3021,6 +3021,73 @@
         Developer: ViewGroupMixIn#getPrimaryView() may return <code>null</code>, and needs to be checked by plugins depending on this version of weekly and beyond.
         It is an intermediate state until a default view is implemented.
 
+- version: 2.204.3
+  date: 2020-02-28
+  changes:
+  - type: bug
+    category: bug
+    pull: 4430
+    issue: 60750
+    authors:
+    - jglick
+    message: |-
+      If the Jenkins root URL has been configured by scripts prior to running the setup wizard, skip the location configuration panel even if selecting the option to skip creation of an admin user.
+  - type: bug
+    category: bug
+    pull: 4433
+    issue: 60725
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent inaccurate warnings about missing classes on Java 11 triggered by JavaMelody when Monitoring Plugin is installed.
+  - type: bug
+    category: bug
+    pull: 4422
+    issue: 60716
+    authors:
+    - jglick
+    message: |-
+      Include details in the system log when a build rotation fails.
+  - type: bug
+    category: bug
+    pull: 4434
+    issue: 60678
+    authors:
+    - MRamonLeon
+    message: |-
+      Fix java version check for AdoptOpenJDK 11.
+  - type: bug
+    pull: 4413
+    issue: 60625
+    message: |-
+      Prevent Jenkins page rendering from being blocked when the update center data parsing is in progress.
+    authors:
+      - jglick
+  - type: bug
+    category: bug
+    pull: 4452
+    issue: 57888
+    authors:
+    - lrpg
+    message: |-
+      Winstone 5.7: Fix support of system logging customization (regression in 2.177).
+    references:
+      - pull: 4452
+      - issue: 57888
+      - issue: 60821
```

Comment:
This would be a separate entry IMHO. https://issues.jenkins-ci.org/browse/JENKINS-60821 has nothing to do with the changelog message

Reply:
Thanks.  See 55608ac6f8727e5784b1d785696ce9724981ce14 for the proposed change.  Since that is an RFE in the weekly changelog, I placed it at the top of the list.

It feels strange to place an `Internal` change at the top of the list, but we've generally preferred to have RFE entries precede bug entries.  Is that OK in this case?

Reply:
Merged on the assumption that it was better to have a potentially flawed changelog visible than to not have a changelog.